report/extra_metadata_templates.go

```diff
@@ -0,0 +1,33 @@
+package report
+
+type ExtraMetadataTemplates []MetadataRowTemplate
```

Comment:
Why "Extra"? Why not all?

Also, why a list and not a map from id -> template?

And merge does not seem entirely commutative.